guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -30,9 +27,42 @@ ifdef::katello,satellite,orcharhino[]
 
 endif::[]
 |====
++
+These values are based on expected use case scenarios and can vary according to individual environments.
+ifdef::katello,satellite[]
+The runtime size was measured with {RHEL} 7, 8, and 9 repositories synchronized.
+endif::[]
+ifdef::katello,orcharhino,satellite[]
+* If you mount the `/tmp` directory as a separate file system, use the `exec` mount option in the `/etc/fstab` file.
++
+If `/tmp` is already mounted with the `noexec` option, change the option to `exec` and remount the file system.
+This is a requirement for the `puppetserver` service to work.
+* If you mount the `/var/lib/pulp` directory as an NFS share, specify the SELinux context of the `/var/lib/pulp` directory in the file system table.
+Add the following lines to `/etc/fstab`:
++
+----
+nfs.example.com:/nfsshare  /var/lib/pulp  nfs  context="system_u:object_r:var_lib_t:s0"  1 2
+----
++
+If the NFS share is already mounted, remount it using the above configuration and restore the SELinux context:
++
+----
+# restorecon -R /var/lib/pulp
+----
++
+* Do not use symbolic links for `/var/lib/pulp/`.
+endif::[]
+ifeval::["{mode}" == "connected"]
+* If you intend to synchronize RHEL content ISOs to {Project}, note that all minor versions of {RHEL} also synchronize.
+Ensure adequate storage on your {Project} to manage this.
+endif::[]
 
 ifdef::foreman-el,katello,satellite[]
 For external database servers: `{postgresql-lib-dir}` with installation size of 100 MB and runtime size of 20 GB.
+endif::[]
```

Comment:
I was thinking of moving this somewhere closer to the external db procedures, like https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_preparing-for-using-external-databases.adoc or https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_installing-postgresql.adoc. Any opinions?